src/feature/control/control_getinfo.c

```diff
@@ -1436,6 +1437,37 @@ getinfo_helper_liveness(control_connection_t *control_conn,
   return 0;
 }
 
+/** Implementation helper for GETINFO: answers queries about shared random
+ * value. */
+STATIC int
+getinfo_helper_rephist(control_connection_t *control_conn,
+                       const char *question, char **answer,
+                       const char **errmsg)
+{
+  (void) control_conn;
+  (void) errmsg;
+  int result = -1;
```

Comment:
I would not assign it here so the compiler can tell us when we use it unassigned. And I don't think we should ever use -1 as for an unknown `question`, an control error code should be returned?